core/model/ft.go

```diff
@@ -0,0 +1,30 @@
+package model
+
+type CreateFTReq struct {
+	DID        string  `json:"did"`
+	FTName     string  `json:"ftname"`
+	FTCount    int     `json:"ftcount"`
+	TokenCount float64 `json:"tokencount"`
+}
+
+type TransferFTReq struct {
+	Receiver   string `json:"receiver"`
+	Sender     string `json:"sender"`
+	FTName     string `json:"FTName"`
+	FTCount    int    `json:"FTCount"`
+	Comment    string `json:"comment"`
+	Type       int    `json:"type"`
```

Comment:
Instead of using type it will be better to use quorum_type because, since type is a reserved word, it can create issue while we write contract and all.

Reply:
Type changed to quorumType.